api/actors.yml

```diff
@@ -0,0 +1,149 @@
+models:
+  Actor:
```

Comment:
While it's bound to rot, it's still probably worthwhile to link to the spec for spec-related things, with an explicit disclaimer that these structures are not intended to match the spec exactly (indeed, they are intended to be more usable), but should generally agree with it.

https://github.com/filecoin-project/specs/blob/master/data-structures.md#actor